docs/authentication/social-connections/coinbase.mdx

```diff
@@ -1,38 +1,78 @@
 
-- You need to create a Clerk Application in your [Clerk Dashboard](https://dashboard.clerk.com/). For more information, check out our [Set up your application guide](/docs/quickstarts/setup-clerk).
-- You need to have a Coinbase developer account. To create one, [click here](https://www.coinbase.com/).
+To make the setup process easier, it's recommended to keep two browser tabs open: one for your [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) and one for your [Coinbase's Developer Platform](https://portal.cdp.coinbase.com).
 
-## Configuring Coinbase social connection
+<Steps>
+  ### Enable Coinbase as a social connection in Clerk
 
-First, you need to create a new OAuth2 Coinbase app. You can find this option in your **Profile > Settings > API > New OAuth2 Application**.
+  1. In the Clerk Dashboard, navigate to the [**SSO Connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page.
+  1. Select **Add connection** and select **For all users**.
+  1. In the **Choose provider** dropdown, select **Coinbase**.
+  1. Ensure that both **Enable for sign-up and sign-in** and **Use custom credentials** are toggled on.
+  1. Save the **Redirect URI** somewhere secure. Keep the modal and page open.
 
-You need to set all the required fields and the corresponding **Permitted Redirect URIs**. Under **Advanced Settings**, make sure to also select **Reuse Existing Tokens** for **Previously-Authorized Users** so that your users won't get notified everytime they sign in with their Coinbase account. Then, click **Create application**.
+  ### Create a Coinbase app
 
-![Creating application](/docs/images/authentication-providers/coinbase/creating-application.jpg)
+  > [!NOTE]
+  > Coinbase automatically creates a default project for you named `Project 1`. Select the icon next to the project name to rename.
 
-Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections). In the top navigation, select **Configure**. Then in the sidebar, select **SSO Connections**. Select the **Add connection** button, and select **For all users**. In the **Choose provider** dropdown, select **Coinbase**. Toggle on **Use custom credentials** and copy **Redirect URI**. Make sure the value matches the **Redirect URIs**, as shown below.
+  1. In your [Coinbase Developer Platform project dashboard](https://portal.cdp.coinbase.com/projects), navigate to the **API Keys** tab.
+  1. Select **OAuth**, then select **Create client**.
+  1. Complete the required fields. In **Redirect URIs**, paste the **Redirect URI** you saved from the Clerk Dashboard.
+  1. Select **Create client**. The page will refresh and you should see the **Client ID** and **Client Secret** in the **OAuth** section. Save both values somewhere secure. Keep this page open.
```

Comment:
they actually never have to come back to this page in the tutorial, so its fine to close it.

```suggestion
  1. Select **Create client**. The page will refresh and you should see the **Client ID** and **Client Secret** in the **OAuth** section. Save both values somewhere secure.
```